tests/ut_common_test_file.py

```diff
@@ -0,0 +1,18 @@
+"""
+The unit test checks if build_component.py works fine.
+"""
+
+import subprocess
+import sys
+
+sub = subprocess.Popen([sys.executable, '../DEPENDENCIES/common/build_component.py', '-t'], 
+                                         stderr=subprocess.PIPE, 
+                                         stdout=subprocess.PIPE)
+(out, err) = sub.communicate()
+
+#should cause test to fail if there's anything on stderr
+if err != '':
+  print "FAIL"
+  
+if not "Done building!" in out:
```

Comment:
You'll likely want to test if the files are actually copied over etc., not only that the script didn't return an error.

BTW, for the natural `utf` way of detecting matches in output of tests, see the `#pragma out`  directive, https://seattle.poly.edu/wiki/UnitTestFramework#StandardOutError